packages/core/test/test-util/test-services/openapi/swagger-yaml-service/test-license

```diff
@@ -0,0 +1,35 @@
+SAP DEVELOPER LICENSE AGREEMENT
```

Comment:
Maybe use a dummy license file (or any other file, doesn't have to have a notion of license), because this might be confusing.
Acutally, adding a changelog is enough in my opinion.

Reply:
I used the `test-license` as the file name, but you are right, the content might also be confusing.